packages/devextreme-scss/scss/widgets/material/chat/_sizes.scss

```diff
@@ -72,10 +89,14 @@ $chat-messagelist-day-header-first-padding-top: 20px !default;
   $chat-errors-row-gap: 2px !default;
   $chat-error-padding-block: 5px !default;
 
-  $chat-messagelist-empty-icon-box-size: 48px !default;;
+  $chat-messagelist-empty-icon-box-size: 48px !default;
   $chat-messagelist-empty-icon-size: 32px !default;
   $chat-messagelist-empty-icon-margin-bottom: 14px !default;
   $chat-messagelist-empty-row-gap: 2px !default;
   $chat-messagelist-empty-message-font-size: 14px !default;
   $chat-messagelist-empty-prompt-font-size: 12px !default;
+
+  $chat-typingindicator-bubble-height: 2.333333em !default;
+  $chat-typingindicator-bubble-width: 5em !default;
```

Comment:
It should be 60x24px instead of 65x30.33px according to the design

Reply:
[97c5030602394817755eb45ba60b230028eb164f]